lib/util.js

```diff
@@ -21,21 +29,20 @@ exports.format = function(f) {
 
   if (arguments.length === 1) return f;
 
+  const len = arguments.length;
+  const args = new Array(len);
+  for (var i = 0; i < len; i++) {
```

Comment:
Sorry, you'll also need to move the `var i` declaration out and above and change `var i = 1;` a few lines below to `i = 1` to avoid a linter error.